pkg/trait/environment.go

```diff
@@ -18,8 +18,11 @@ limitations under the License.
 package trait
 
 import (
+	"fmt"
 	"os"
+	"strings"
 
+	"github.com/magiconair/properties"
```

Comment:
We should try not to depend on this library any longer. It could have been find for CLI, but the operator should limit the exposure to third party libraries.